packages/udaru-core/lib/ops/validation.js

```diff
@@ -1,13 +1,13 @@
 'use strict'
 
-const Joi = require('joi')
+const Joi = require('@hapi/joi')
 
 const requiredString = Joi.string().required()
 const requiredStringId = Joi.string().regex(/^[A-Za-z0-9_-]+$/).required().max(128)
 const MetaData = Joi.object().optional().description('Metadata').label('MetaData')
 
 const PolicyIdString = requiredStringId.description('Policy Id String').label('PolicyIdString')
-const PolicyVariables = Joi.object().optional().pattern(/^(?!(udaru)|(request)).*$/igm, requiredString).description('A list of the variables with their fixed values').label('variables')
+const PolicyVariables = Joi.object().optional().pattern(/^(?!(udaru)|(request)).*$/im, requiredString).description('A list of the variables with their fixed values').label('variables')
```

Comment:
Why this change?

Reply:
It's a breaking change when upgrading to v14. Previously the `/g` was ignored, but now `joi` will just crash. I've gone through tests to check if the change would hurt. This is the only place the problem occurs, and it looks to me we can make this change without any consequences.

Reply:
I'll have a deeper look into that change again. I may have overlooked something.